src/features/orders/Orders.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import {View, Text, Image, ListRenderItemInfo, TouchableOpacity, ImageProps} from 'react-native';
+import {RowMap, SwipeListView} from 'react-native-swipe-list-view';
+import SwipeIcon from 'react-native-vector-icons/MaterialIcons';
+import HeartIcon from 'react-native-vector-icons/FontAwesome5';
+
+import {Counter} from './Counter';
+import {CustomButton} from 'components/custom_button';
+import Food1 from '@assets/image/food.png';
+import Food2 from '@assets/image/food2.png';
+import Food3 from '@assets/image/food3.png';
+import {styles} from './styles/orders';
+
+interface State {
+  carts: Array<Item>;
+}
+
+interface Item {
```

Comment:
@DiamondRock597 We using this model in State interface, lets initialise Item interface and only after that use it.